app/assets/stylesheets/partials/_index.scss

```diff
@@ -67,6 +69,17 @@
     white-space: nowrap;
   }
 
+  .chevron-icon {
+    margin: 10px;
+  }
+
+  .header-breadcrumbs {
+    display: flex;
```

Comment:
Properties should be ordered align-items, display, flex-direction, justify-content